core/scripts/chaincli/command/keeper/verifiable_load.go

```diff
@@ -0,0 +1,20 @@
+package keeper
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/config"
+	"github.com/smartcontractkit/chainlink/core/scripts/chaincli/handler"
+)
+
+// verifiableLoad represents the command to get verifiable load testing details
+var verifiableLoad = &cobra.Command{
+	Use:   "verifiable-load",
+	Short: "Print verifiable load testing details to console",
+	Long:  `Print verifiable load testing details to console, including details of every active upkeeps and total result`,
```

Comment:
`Print verifiable load testing details to console, including details of every active upkeep and total results` 🏄

Reply:
done